JSON/mv-params-input.json

```diff
@@ -0,0 +1,17 @@
+{
+    "common": [
+    {
+        "name": "global",
+        "params": [
+        { "arg": "bs", "vals": [ "4k", "8k" ] },
+        { "arg": "rw", "vals": [ "read", "write" ] }
+        ]
+    }
+    ],
+    "sets": [
+    [
+        { "common": "global" },
+        { "arg": "ioengine", "vals": [ "sync" ] }
+    ]
+    ]
```

Comment:
Can we try to maintain a hierarchy via indentation?  I just think it makes it easier to read...

Reply:
I'm still trying to get your point here, could you please be more specific ?